### PR TITLE
feature/AT-8020-IGCE-page-1a-cannot-proceed

### PR DESCRIPTION
**Pull request by tonysumner-ccpo**

### CODE REVIEW

File: src/steps/10-FinancialDetails/IGCE/CannotProceed.vue [hunk 1]
```diff
@@ -1,15 +1,73 @@
 <template>
-  <div>
-    Future page for Cannot Proceed
-  </div>
+  <v-container fluid class="container-max-width mb-7">
+    <v-row>
+      <v-col class="col-12">
+        <h1 class="page-header mb-3">
+          To accurately complete your price estimate, you need to revisit the
+          Contract Details and Performance Requirements sections
```
Comment (ericyoungquist-ccpo) [inline src/steps/10-FinancialDetails/IGCE/CannotProceed.vue:7]: AC 1 - need logic to show different ends to the title here based on what's missing

File: src/steps/10-FinancialDetails/IGCE/CannotProceed.vue [hunk 2]
```diff
@@ -1,15 +1,73 @@
...
+  components: {
+    ATATAlert,
+  },
 })
 export default class CannotProceed extends Vue {
+
+  get isPoPIncomplete(): boolean{
+    if (Periods.periods){
+      return Periods.periods.length === 0; 
+    } else {
```
Comment (ericyoungquist-ccpo) [inline src/steps/10-FinancialDetails/IGCE/CannotProceed.vue:63]: don't need the `else` here... or better yet, one line would do it 
`return Periods.periods !== null && Periods.periods.length > 0;`

File: src/router/resolvers/index.ts [hunk 3]
```diff
@@ -558,6 +558,40 @@ export const DowSummaryPathResolver = (current: string, direction: string): stri
...
+    }
+  }
+  return routeNames.CannotProceed;
+};
+
+export const IGCEGatherPriceEstimatesResolver = (current: string): string => {
+  const hasLegitPeriods =  Periods.periods && Periods.periods.length > 0;
+  const isCompleteDOW = DescriptionOfWork.isIncomplete === false;
+  const validToProceed = hasLegitPeriods && isCompleteDOW;
+
```
Comment (ericyoungquist-ccpo) [inline src/router/resolvers/index.ts:580]: need to add this for when using Back button - if it's on Travel Estimates and clicking back, it's skipping the Gather Price Estimates page and going straight to the first page
Screen Shot 2022-10-05 at 2 45 58 PM

**Review by ericyoungquist-ccpo — CHANGES_REQUESTED** [review]

File: src/steps/10-FinancialDetails/IGCE/CannotProceed.vue (hunk 2 above)
Comment (tonysumner-ccpo) [reply src/steps/10-FinancialDetails/IGCE/CannotProceed.vue:63]: `return Periods.periods !== null && Periods.periods.length > 0;` will return isPopIncomplete===false (POP is complete);

If `periods` is null or undefined, isPopIncomplete should return true.

File: src/steps/10-FinancialDetails/IGCE/CannotProceed.vue (hunk 1 above)
Comment (tonysumner-ccpo) [reply src/steps/10-FinancialDetails/IGCE/CannotProceed.vue:7]: done.

File: src/steps/10-FinancialDetails/IGCE/CannotProceed.vue [hunk 4]
```diff
@@ -1,15 +1,73 @@
...
+              In order to start the Requirements Cost Estimate section, you need
+              to:
+            <ul>
+              <li v-show="isPoPIncomplete" class="my-2">
+                <a href="/#/period-of-performance/period-of-performance">
+                  Complete your period of performance
+                </a>
+              </li>
+              <li v-show="isPerformanceReqsIncomplete">
+                <a href="/#/performance-requirements/">
```
[the remark this answers is not in the document]
Comment (tonysumner-ccpo) [reply src/steps/10-FinancialDetails/IGCE/CannotProceed.vue:34]: done

File: src/router/resolvers/index.ts (hunk 3 above)
Comment (tonysumner-ccpo) [reply src/router/resolvers/index.ts:580]: Done, thx,.

**Review by ericyoungquist-ccpo — APPROVED** [review]

**Pull request merged**